### PR TITLE
test display with cypress

### PR DESCRIPTION
**Pull request by nicolerue**
# PR Review for Rancid Tomatillos 

## Summary
This Pull Request (PR) includes several significant updates to our project. Here's a breakdown of the changes:

- [x] Refactored fetch requests to handle errors.
- [x] Improved CSS organization by moving most styles to a separate SCSS file.
- [x] Enhanced the modal component by refactoring the `useEffect` hook to handle two side effects: fetching movie data and fetching movie video objects.
- [x] Installed an npm package to facilitate video display within the modal.
- [x] Added a Cypress test for our display, following the recommended query -> query -> command -> assertion flow, and the test passes!
- [x] Kept the GitHub project board up to date.

#### Thanks for your feedback Nick, all feedback is super appreciated, we thought of the following specific things we would appreciate feedback on for improvement: 
- [ ] **Error Handling in Fetch Requests:** 
- [ ] **Cypress Test and Testing Coverage:**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -10,35 +10,37 @@ function App() {
...
+  const [selectedMovieTrailerLink, setSelectedMovieTrailerLink] = useState("");
+  const [error, setError] = useState(null);
 
- 
   function getMoviesFromApi() {
     fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
       .then((res) => {
         if (!res.ok) {
-          throw new Error("Oops! Something went wrong on the server. Please try again later.");
+          throw new Error(
```
Comment (nicktu12) [inline src/App.js:20]: Are you planning to throw errors, or use state to show errors to your user? I would pick on and stay consistent with it!

File: src/Components/Modal.js [hunk 2]
```diff
@@ -1,28 +1,49 @@
...
+  function getSingleMovieVideoApi() {
+    fetch(
+      `https://rancid-tomatillos.herokuapp.com/api/v2/movies/${selectedMovieID}/videos`
+    )
+      .then((res) => res.json())
+      .then((data) => {
+        const trailer = data.videos.find((video) => {
+          return video.type === "Trailer";
+        });
+        console.log(trailer);
```
Comment (nicktu12) [inline src/Components/Modal.js:37]: I would encourage you to add a task to your project board to remove any console.logs from your project - so you don't forget!

File: src/Components/Modal.js [hunk 3]
```diff
@@ -1,28 +1,49 @@
...
+        const trailerKey = trailer.key;
```
Comment (nicktu12) [inline src/Components/Modal.js:38]: This variable assignment might not be necessary - just pass `trailer.key` directly into the method below if you don't need this variable for more than one thing!

File: src/Components/Modal.js [hunk 4]
```diff
@@ -35,10 +56,10 @@ function Modal({
         <div
           className="backdrop-image"
           style={{
-            display: 'flex',
-            gap: '5rem',
-            color: 'white',
-            height: '100vh',
+            display: "flex",
```
Comment (nicktu12) [inline src/Components/Modal.js:59]: Any reason for inline styling over external css here?

**Review by nicktu12 — COMMENTED** [review]
Left a few comments for yall!

**Review by Jnguyen615 — COMMENTED** [review]
I like how we managed to get the video to display for the user after some trial and error. The comments from Nick are definite improvements for us to work on.

**Pull request merged**